libsolidity/interface/CompilerStack.cpp

```diff
@@ -1248,6 +1248,15 @@ void CompilerStack::generateIR(ContractDefinition const& _contract)
 	if (!compiledContract.yulIR.empty())
 		return;
 
+	solAssert(_contract.sourceUnit().annotation().useABICoderV2.set(), "");
```

Comment:
The `*` below already asserts that it is set.

Reply:
Good catch, removed.